test/jdk/tools/jpackage/windows/WinInstallerIconTest.java

```diff
@@ -67,31 +66,23 @@ public void test() throws IOException {
         BufferedImage[] defaultInstallerIconImg = new BufferedImage[1];
 
         // Create installer with the default icon
-        createInstaller(null, "WithDefaultIcon", installerIconImg -> {
-            defaultInstallerIconImg[0] = installerIconImg;
-        }, null, null);
-
-        BufferedImage[] customInstallerIconImg = new BufferedImage[1];
+        long size1 = createInstaller(null, "WithDefaultIcon");
 
         // Create installer with custom icon.
-        // This installer icon should differ from the icon
-        // of the installer created with the default icon.
-        createInstaller(customIcon, "2", installerIconImg -> {
-            customInstallerIconImg[0] = installerIconImg;
-        }, null, defaultInstallerIconImg[0]);
-
-        // Create installer with custom icon again.
-        // This installer icon should differ from the icon
-        // of the installer created with the default icon and should have
-        // the same icon as the icon of installer created with custom icon.
-        createInstaller(customIcon, null, null,
-                customInstallerIconImg[0], defaultInstallerIconImg[0]);
+        long size2 = createInstaller(customIcon, "WithCustom2Icon");
+
+        // Create another installer with custom icon.
+        long size3 = createInstaller(customIcon, "WithCustom3Icon");
```

Comment:
The test should produce WinInstallerIconTest-1.0.exe artifact referenced in SQE test spec. But this patch will not produce it. It will produce WinInstallerIconTestWithDefaultIcon-1.0.exe, WinInstallerIconTestWithCustom2Icon-1.0.exe and WinInstallerIconTestWithCustom3Icon-1.0.exe artifacts and all of them will be deleted.

Reply:
yes - I was changing the names to be the same length in attempt to create repeatable exe size - but that doesn't matter, so will restore the suffix's to what they were so WinInstallerIconTest-1.0.exe  will be left for JPKG001-024 in Test Spec.

Reply:
done - name change and test reverted so remaining artifact is WinInstallerIconTest-1.0.exe (and it contains the custom icon)